### DISCUSSION TITLE
公式编号中，章节编号与公式编号的连接符设置

### DISCUSSION THREAD

**Discussion by dreistein543** [nju-lug/NJUThesis#134] [Q&A] [answered]
目前默认的连接符是一个句点 `例如(1.1) (2.1) (2.2)` ，这种链接方式对于某些专业来说是合适的。

但是也有些专业更倾向于使用短横线链接 `例如 (1-1) (2-1) (2-2)` 这种方式。

希望能加入设定，或者在手册中介绍一下如何修改连接符

**Comment by atxy-blip** [comment, marked as answer]
请看 <https://tex.stackexchange.com/questions/61756/how-to-change-equation-numbering-style>

原回答者给的两种方法在 `njuthesis` 中均有效。

**Reply by dreistein543** [reply]
另外，如果想对全局（图、表、公式）都使用横线连接符，也是类似的做法，如下
可以通过修改 `\thechapter{}`和`\arabic{}`中间的符号来自定义连接符
```latex
\usepackage{amsmath}
\renewcommand{\thefigure}{\thechapter{}-\arabic{figure}}       % 图
\renewcommand{\thetable}{\thechapter{}-\arabic{table}}         % 表
\renewcommand{\theequation}{\thechapter{}-\arabic{equation}}   % 公式
```
